### DISCUSSION TITLE
Wont let me delete repository

### DISCUSSION THREAD

**Discussion by jamesrcutmore** [community/community#130761] [Repositories]
### Select Topic Area

General

### Body

Hi everyone, Iam new here and looking to ask a question, My github will not let me delete a repository, does not seem to recognise the name when I  enter the box. 
![Screenshot (448)](https://github.com/community/community/assets/95052094/0bb4917d-da56-4d73-bd36-7ff84941863b)

**Comment by davevad93** [comment]
Hi @jamesrcutmore ,

Simply delete the quotation marks and you will be able to delete the repo.  `jamesrcutmore/jamesrcutmore`

**Reply by jamesrcutmore** [reply]
Thank you. All done 😁

**Reply by davevad93** [reply]
You're welcome @jamesrcutmore ! 👍
